build/scripts/tests.sh

```diff
@@ -20,10 +20,15 @@ xunit_console_version="$(get_package_version dotnet-xunit)"
 
 if [[ "${runtime}" == "dotnet" ]]; then
     target_framework=netcoreapp2.0
+    dir_list="${unittest_dir}"/*/netcoreapp2.0/*.UnitTests.dll
     xunit_console="${nuget_dir}"/dotnet-xunit/"${xunit_console_version}"/tools/${target_framework}/xunit.console.dll
 elif [[ "${runtime}" == "mono" ]]; then
     source ${root_path}/build/scripts/obtain_mono.sh
-    target_framework=net461
+    unittest_dll_list=(
+        "${unittest_dir}/CSharpCompilerSymbolTest/net461/Roslyn.Compilers.CSharp.Symbol.UnitTests.dll"
+        "${unittest_dir}/CSharpCompilerSyntaxTest/net461/Roslyn.Compilers.CSharp.Syntax.UnitTests.dll"
```

Comment:
I'm guessing this is the static list of what dlls were actually running (positive list instead of negative list). Good to keep in mind for the future.